### ISSUE TITLE
Profiling utility profile.py doesn't work correctly with multiple osquery tables

### ISSUE THREAD

**Issue by ar-mi** [osquery/osquery#6907]
# Bug report

### What operating system and version are you using?

```
version = 10.15.7
build = 19H114
platform = darwin
```
### What version of osquery are you using?

`version = 4.6.0`
### What steps did you take to reproduce the issue?
1) Created test config with scheduled queries (test.conf):
```
{
  "schedule": {
    "proc1": {
      "query": "SELECT * FROM system_info",
      "interval": 60
    },
    "proc2": {
      "query": "SELECT * FROM osquery_info",
      "interval": 60
    },
    "proc3": {
      "query": "SELECT * FROM users",
      "interval": 60
    }
  }
}
```
2) Cloned your repository using command `git clone https://github.com/osquery/osquery.git`
3) Launched query profiling tool profile.py using command `./tools/analysis/profile.py --shell $(which osqueryi) --config test.conf --rounds 3 --count 10` and got these results: 
<img width="1149" alt="Снимок экрана 2021-01-16 в 16 10 55" src="https://user-images.githubusercontent.com/75211836/104812762-81ca2c00-5815-11eb-8be8-80375c97bb44.png">
4) Checked these queries (system_info, osquery_info) manually through osqueryi and got these results:
<img width="435" alt="Снимок экрана 2021-01-16 в 16 00 14" src="https://user-images.githubusercontent.com/75211836/104812583-3bc09880-5814-11eb-9290-4aec6fd5621f.png">

### What did you expect to see?
I wanted to test performance of my complex queries through the profile.py because of its periodic hitting in the denylist on users, but I ran into the behavior described above. I have not tested all Osquery tables on Mac, only the ones in my query. 
When correcting, it is better to check the fix on all tables on all operating systems.

### What did you see instead?
Inability to test the performance of queries using several Osquery tables.

**Comment by JohnSpeno** [comment]
Yeah, I see this too and so far found it to be true for `osquery_info` and `system_info`. Looks like the errors come from `osqueryi` itself. E.g:

```
07:01 speno@charm:~
: %; /usr/local/bin/osqueryi --profile 1 'select 1 from system_info;'
E0304 07:01:29.147604 205944320 virtual_table.cpp:1006] Exception while executing table system_info: Cannot get database value: host_uuid_v3
Query failed (1): Cannot get database value: host_uuid_v3
```
and
```
07:01 speno@charm:~
: %; /usr/local/bin/osqueryi --profile 1 'select 1 from osquery_info;'
E0304 07:02:00.127287 354088448 virtual_table.cpp:1006] Exception while executing table osquery_info: Cannot get database value: failed_queries
Query failed (1): Cannot get database value: failed_queries
```

**Comment by directionless** [comment]
Thanks for reporting this issue with the `profile.py` profiling utility. This issue has been open for over 5 years without activity, and both the macOS version (10.15) and osquery version (4.6.0) referenced are quite outdated at this point.

We're going to close this for now due to inactivity. If you're still experiencing issues with the profiling tool on a current version of osquery, please feel free to open a new issue with updated reproduction steps and we'll be happy to take a look.

osquery is a volunteer project, and contributions to improve the profiling tooling are always welcome!
